interfaces/cython/cantera/yaml2ck.py

```diff
@@ -0,0 +1,728 @@
+        break_on_hyphens=False,
+    )
+    if "description" in solution.input_header:
+        header_text = header_wrapper.fill(
+            solution.input_header["description"],
+        ) + "\n"
+    else:
+        header_text = header_wrapper.fill("") + "\n"
+    mechanism_text = [
+        header_text,
+        build_elements_text(
+            all_elements,
+        ),
+        build_species_text(
+            all_species,
+        ),
+    ]
+    if thermo_path is None:
+        mechanism_text.append(
+            build_thermodynamics_text(
+                all_species,
+                separate_file=False,
+            ),
+        )
+    else:
+        thermo_text = [
+            header_text,
+            build_thermodynamics_text(
+                all_species,
+                separate_file=True,
+            ),
+        ]
+        thermo_path.write_text("\n".join(thermo_text))
+
+    # TODO: Handle phases without reactions
+    all_reactions = solution.reactions()
+    mechanism_text.append(
+        build_reactions_text(all_reactions)
+    )
+
+    if transport_path is None and transport_exists:
+        mechanism_text.append(
+            build_transport_text(all_species, separate_file=False)
+        )
+    elif transport_path is not None and transport_exists:
+        transport_text = [
+            header_text,
+            build_transport_text(all_species, separate_file=True),
+        ]
+        transport_path.write_text("\n".join(transport_text))
+
+    mechanism_path.write_text("\n".join(mechanism_text))
+
+    return output_files
+
+
+def main():
+    """
+    Parse command line arguments and pass them to `convert`
+    """
+
+    parser = argparse.ArgumentParser(
+        description="Convert Cantera YAML input files to Chemkin-format mechanisms",
+    )
+
+    parser.add_argument("input", help="The input YAML filename. Required.")
+    parser.add_argument(
+        "--phase-name", help="Name of the phase to load from the input."
+    )
+    parser.add_argument("--mechanism", help="The output mechanism filename.")
+    parser.add_argument("--thermo", help="The output thermodynamics filename.")
+    parser.add_argument("--transport", help="The output transport filename.")
+    parser.add_argument(
+        "--sort-elements",
+        choices=[None, "alphabetical", "molecular-weight"],
+        default=None,
+        help=("Sort elements in source order (None), alphabetically, or from lowest to "
+              "highest atomic mass."),
+    )
+    parser.add_argument(
+        "--sort-species",
+        choices=[None, "alphabetical", "molecular-weight"],
+        default=None,
+        help=("Sort species in source order (None), alphabetically, or from lowest to "
+              "highest molecular weight."),
+    )
+    parser.add_argument(
+        "--overwrite",
+        choices=[True, False],
+        default=False,
+        type=bool,
+        help="Overwrite existing output files.",
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "--overwrite",
        action="store_true",
        default=False,
        help="If set, overwrite existing output files.",
    )
```
Same for `--validate` below.

Together with the other suggestions for `metavar` (and moving `--overwrite` and `--validate` up, the formatting of `yaml2ck --help` becomes a lot easier to parse visually, i.e.
```
(cantera-dev) PS C:\Users\ischo\GitHub\cantera> yaml2ck --help
usage: yaml2ck [-h] [--phase-name PHASE] [--mechanism MECH] [--thermo THERMO] [--transport TRAN] [--overwrite]
               [--validate] [--sort-elements {None,alphabetical,molecular-weight}]
               [--sort-species {None,alphabetical,molecular-weight}]
               input

Convert Cantera YAML input files to Chemkin-format mechanisms

positional arguments:
  input                 The input YAML filename. Required.

options:
  -h, --help            show this help message and exit
  --phase-name PHASE    Name of the phase to load from the input.
  --mechanism MECH      The output mechanism filename.
  --thermo THERMO       The output thermodynamics filename.
  --transport TRAN      The output transport filename.
  --overwrite           If set, overwrite existing output files.
  --validate            If set, check that the mechanism can be loaded back into Cantera.
  --sort-elements {None,alphabetical,molecular-weight}
                        Sort elements in source order (None), alphabetically, or from lowest to highest atomic mass.
  --sort-species {None,alphabetical,molecular-weight}
                        Sort species in source order (None), alphabetically, or from lowest to highest molecular
                        weight.
```